src/Operations.jl

```diff
@@ -903,7 +903,7 @@ function rm(ctx::Context, pkgs::Vector{PackageSpec})
         uuid ∉ drop
     end
     if length(ctx.env.project.deps) == n
-        println(ctx.io, "No changes")
+        println(ctx.io, "No changes to package dependencies")
```

Comment:
What about: `"no direct dependencies have been removed"`?

Reply:
That does seem more accurate! Updated. Thanks